python/HelloWorldTest.py

```diff
@@ -0,0 +1,76 @@
+from selenium import webdriver
+import unittest
+from pathlib import Path
+import os
+from sys import platform as _platform
+
+from selenium.webdriver.chrome.options import Options
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+import re
+
+
+class HelloWorldTests (unittest.TestCase):
+    TEN_SECOND = 10
+
+    RIGHT_NAME_INPUT_ID = "et_pb_contact_name_1"
+    RIGHT_MESSAGE_INPUT_ID = "et_pb_contact_message_1"
+    EQUATION_TEXT_CLASS = "et_pb_contact_captcha_question"
+    RIGHT_ANSWER_INPUT_CSS = ".input.et_pb_contact_captcha"
+    RIGHT_SUBMIT_BUTTON_CSS = "div#et_pb_contact_form_1 button"
+    MESSAGE_TEXT_CSS = "div#et_pb_contact_form_1 div.et-pb-contact-message"
+
+    def setUp(self):
+        # It assumes chrome driver is saved at <user home>/drivers/chromedriver
+        user_home = str(Path.home())
+        driver_path = r"drivers"
+        driver_name = r"chromedriver.exe" if _platform == "Windows" else r"chromedriver"
+        webdriver_path = os.path.join(user_home, driver_path, driver_name)
+        chrome_options: Options = webdriver.ChromeOptions()
+        prefs = {"profile.default_content_setting_values.notifications": 2}
+        chrome_options.add_experimental_option("prefs", prefs)
+        self.driver = webdriver.Chrome(executable_path=webdriver_path, chrome_options=chrome_options)
+
+    def test_verify_captcha(self):
+        # Go to the site
+        self.driver.get("https://www.ultimateqa.com/filling-out-forms/")
```

Comment:
`self.driver` is used in many places across this test, maybe it would be worth to change it to get an alias this way:
`driver = self.driver`

Then you just use `driver.get()`, `driver.find_element()` and so on and so forth.

Reply:
updated.